tools/CAT/src/components/Button/index.jsx

```diff
@@ -0,0 +1,10 @@
+export default function Button({children, className = '', onClick}){
+	return(
+		<button 
+			className={`text-xl rounded-sm bg-activeItems cursor-pointer p-2 ${className}`}
+			onClick={onClick}
+		>
+			{children}
+		</button>
+	)
+}
```

Comment:
The same here 

And u can use another interface

`ButtonProps` for ex with className and onClick props

Ahhh, `ButtonProps `should be used in `AddButton `too